pom.xml

```diff
@@ -404,6 +406,31 @@ under the License.
                 </property>
             </activation>
         </profile>
+
+        <profile>
+            <id>hadoop2</id>
+            <properties>
+                <hadoop.version>${hadoop2.version}</hadoop.version>
+            </properties>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>hadoop2</name>
+                </property>
+            </activation>
+        </profile>
+
+        <profile>
+            <id>hadoop3</id>
```

Comment:
Why we need to add hadoop3?

Reply:
There may be an error, perhaps caused by my environment problem, this PR can be closed.